### ISSUE TITLE
About the model in main.py

### ISSUE THREAD

**Issue by wangdtfyguhjknjbhvgc** [nku-zhichengzhang/WECL#8]
Hello! Thank you very much for the code. I see that there are questions earlier about the input of the model without audio streaming, should it be in main.py
def main():
    opt = parse_opts()
    opt.device_ids = list(range(device_count()))
    local2global_path(opt)
    model, parameters = generate_visual_Erase_model(opt)
Should generate_visual_Erase_model(opt) be replaced with generate_vaaerase_model(opt) here? But the accuracy of my model after training with generate_vaaerase_model is less than 30%, does it have anything to do with n_threads? I changed n_threads in opts to 0 due to my lack of memory.

**Comment by nku-zhichengzhang** [comment]
It seems like there might be an issue during your training process. I recommend checking the differences between your setup and our trained model checkpoint. You can find our trained model [ckpt](https://drive.google.com/drive/folders/1YYRkBlpaJRicUc5pWsbFlCETozISgnKy?usp=sharing).

Additionally, setting n_threads to 0 might have impacted the performance, as it could limit data loading efficiency. Consider adjusting this value if possible, depending on your system's memory capacity.

**Comment by wangdtfyguhjknjbhvgc** [comment]
Thank you so much for responding so quickly to my question! So the generate_vaaerase_model function should indeed be used instead of generate_visual_Erase_model in what I think is main.py?
'''
def main():
    os.environ['PYTORCH_CUDA_ALLOC_CONF'] = 'expandable_segments:True'
    opt = parse_opts()
    opt.device_ids = list(range(device_count()))
    local2global_path(opt)
    # 确保模型只加载一次
    print("Loading pretrained model...")
    #model, parameters = generate_visual_Erase_model(opt)
    model, parameters = generate_vaaerase_model(opt)
'''
I will refer to the ve8.pth you provided and increase my n_threads as much as possible to try it! Good luck with your research!
